pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pipeline-ai"
-version = "2.3.2"
+version = "2.3.3"
```

I would probably make this a [minor update rather than a patch](https://semver.org/#summary) as you're adding new functionality
```suggestion
version = "2.4.0"
```